src/main/java/net/pms/newgui/LooksFrame.java

```diff
@@ -320,6 +322,76 @@ public LooksFrame(AutoUpdater autoUpdater, PmsConfiguration configuration) {
 			UIManager.put("Tree.font", sf);
 			UIManager.put("Spinner.font", sf);
 		}
+					// *** Shared Fonts
+					Integer twelve = new Integer(12);
```

Comment:
`new Integer()` is deprecated in Java 9+. It is recommended to use static `Integer.valueOf()` instead which internally returns `new Integer(Value)`.